ferrilata-devilline/Startup.cs

```diff
@@ -1,27 +1,37 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
+﻿using Microsoft.Extensions.Configuration;
 using ferrilata_devilline.Repositories;
 using Microsoft.AspNetCore.Builder;
 using Microsoft.AspNetCore.Hosting;
 using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.EntityFrameworkCore;
 using Microsoft.Extensions.DependencyInjection;
+using System;
 
 namespace ferrilata_devilline
 {
     public class Startup
     {
+
+        private IConfiguration Configuration;
+        public Startup(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+
         // This method gets called by the runtime. Use this method to add services to the container.
         // For more information on how to configure your application, visit https://go.microsoft.com/fwlink/?LinkID=398940
         public void ConfigureServices(IServiceCollection services)
         {
             services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_2);
 
+            string connection = Environment.GetEnvironmentVariable("ConnectionStringsDevillineDB");
```

Comment:
You are addressing an env variable called "ConnectionStringsDevillineDB", but I can't see such variable in your launchSettings.json. If it's configured in your system variables, then it seems that everything that's in  the .json is redundant?

Reply:
line 28 is obsolete, all env vars are setup in terminal with sudo nano ~/.bash_profile

Reply:
this is obsolete, I am deliting it